pkg/operator/csi/credentialsrequestcontroller/credentials_request_controller.go

```diff
@@ -60,6 +65,52 @@ func NewCredentialsRequestController(
 	)
 }
 
+func (c CredentialsRequestController) sync(ctx context.Context, syncContext factory.SyncContext) error {
+	_, status, _, err := c.operatorClient.GetOperatorState()
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+		return err
+	}
+
+	cr, err := c.syncCredentialsRequest(status, syncContext)
+	if err != nil {
+		return err
+	}
+
+	isCredentialsProvisioned, err := isProvisioned(cr)
+	if err != nil {
+		return err
+	}
+
```

Comment:
@jsafrane: this is a middle ground solution between _not checking if cloud-credential-operator created the secret_ and _having a useful message in the co/storage CR_. This should work for [this case](https://bugzilla.redhat.com/show_bug.cgi?id=1867907). What do you think?

Reply:
Yes, it looks like a good compromise.

Reply:
Added tests, PTAL